### DISCUSSION TITLE
Integrating CITE-seq data set with different number of antibodies used in ADT-assay.

### DISCUSSION THREAD

**Discussion by SathiyaNManivannan** [satijalab/seurat#5262] [Q&A]
Hello,

We are interested in integrating a couple of CITE-seq datasets which were run with different number of antibodies for the antibody capture assay. The first experiment was run with 11 antibodies to track cell surface protein expression and the second one was run with 30 antibodies. The 11 antibodies used in the first case are also found in the second case. 

When we ran a default integrative analysis using SCTransform -> Integrate -> Run WNN reduction pipelines, we get huge differences in the wnn reduction heavily influenced by the differences in the number of antibodies. This is presumably due to the integrated assay declaring the expression of missing 19 antibodies to be zero.

To remedy this we tried setting the variable features in the assay to only the common 11 antibodies. This did not change the clustering of cell types into separate clusters in wnn reduction.

Alternatively, we restricted the entire ADT assay to the 11 antibodies are the features. This again resulted in cluster in wnn heavily influenced by ADT assay disparity.

Is there a way to perform ADT analysis where differing number of antibodies is used, without losing the information from the antibodies that are not represented in all the assays? Is different number of antibodies were used, is there a way to perform batch correction for the ADT assay?

**Comment by lucygarner** [comment]
I have a similar question. I have two datasets with a different number of antibodies, but not all of the antibodies in one dataset are found within the other.  Some of the antibodies also differ in clone across the two panels.  What would be the best approach to integrate this data?
